src/pages/tug/apply/index.page.js

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+
+import Apply from './Apply.component';
+import Error from '../../_error.page';
+import { AuthContext } from '~/context';
+import { CommunityHead } from '~/components';
+
+const PageContent = () => {
+  const { isAnonymous } = useContext(AuthContext);
+
+  const isTugMember = false;
+  if (!isAnonymous) {
+    // login();
+    // return null;
+  }
+
+  if (isTugMember) {
+    return <Error statusCode={409} errorMsg={'已经提交过申请并且正在审核中，或者已经加入 TUG 了'} />;
+  } else {
+    return <Apply />;
+  }
+};
+
+const title = 'TiDB User Group 会员申请';
+const description =
+  '加入 TUG，让数据库、大数据从业者找到自己的圈子；' +
+  '探索技术问题，随时随地交流成长，解决问题；发表技术见解，收获前沿知识，提升个人影响力。';
+
+const Page = () => (
```

Comment:
For the new page, do you want to apply a feature toggle before it can be formally released?
Like what I did for the homepage in https://github.com/pingcap/tug-website/commit/36e6be940be8c9fa74dff66c5fa5308306c51d51#diff-ae820e86ad0e3584f74030b82e1f65f8f3beb6b6939b448cd136a13fd1bd59a3